source/backend/api/Helpers/Extensions/PrincipalExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Security.Claims;
+using Pims.Dal.Entities;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Repositories;
+
+namespace Pims.Core.Extensions
+{
+    public static class PrincipalExtensions
+    {
+        public static void HasAccessToAcquisitionFile(this ClaimsPrincipal principal, IUserRepository userRepository, PimsAcquisitionFile acquisitionFile)
```

Comment:
Preference for Throw * naming. In my mind a Has* function would return a boolean.

Reply:
Renamed